docs/nl/docs/index.md

```diff
@@ -0,0 +1,494 @@
+# FastAPI
+
+<style>
+.md-content .md-typeset h1 { display: none; }
+</style>
+
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI framework, zeer goede prestaties, eenvoudig te leren, snel te programmeren, klaar voor productie</em>
+</p>
+<p align="center">
+<a href="https://github.com/fastapi/fastapi/actions?query=workflow%3ATest+event%3Apush+branch%3Amaster" target="_blank">
+    <img src="https://github.com/fastapi/fastapi/workflows/Test/badge.svg?event=push&branch=master" alt="Test">
+</a>
+<a href="https://coverage-badge.samuelcolvin.workers.dev/redirect/fastapi/fastapi" target="_blank">
+    <img src="https://coverage-badge.samuelcolvin.workers.dev/fastapi/fastapi.svg" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/pyversions/fastapi.svg?color=%2334D058" alt="Supported Python versions">
+</a>
+</p>
+
+---
+
+**Documentatie**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**Broncode**: <a href="https://github.com/fastapi/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI is een modern, snel (zeer goede prestaties), web framework voor het bouwen van API's in Python gebruikmakend van standaard Python type hints.
+
+De belangrijkste kenmerken zijn:
+
+* **Snel**: Zeer hoge prestaties, vergelijkbaar met **NodeJS** en **Go** (met dank aan Starlette en Pydantic). [Een van de snelste beschikbare Python frameworks](#prestaties).
+* **Snel te programmeren**: Verhoog de snelheid om functionaliteit te ontwikkelen met ongeveer 200% tot 300%. *
+* **Minder bugs**: Verminder ongeveer 40% van de door mensen (ontwikkelaars) veroorzaakte fouten. *
+* **Intuïtief**: Buitengewoon goede ondersteuning voor editors. <abbr title="ook bekend als automatisch aanvullen, automatisch aanvullen, IntelliSense">Overal automische code aanvulling</abbr>. Minder tijd kwijt aan debuggen.
+* **Eenvoudig**: Ontworpen om gemakkelijk te gebruiken en te leren. Minder tijd nodig om documentatie te lezen.
+* **Kort**: Minimaliseer codeduplicatie. Meerdere functionaliteit voor elke parameterdeclaratie. Minder bugs.
```

Comment:
`Meerdere functionaliteit voor elke parameterdeclaratie`.  Wat bedoel je daar precies mee?

Reply:
Dat een enkele parameterdeclaratie ervoor zorgt dat type kan gecheckt worden, dat de documentatie juist is etc - dus dat je alles maar één keer moet declareren om al deze functionaliteit te bekomen. Het Engels is eigenlijk niet veel informatiever:
> Multiple features from each parameter declaration.

Ik heb het herschreven naar 
> Elke parameterdeclaratie ondersteunt meerdere functionaliteiten.

Maar sta open voor andere alternatieven 🤷

Reply:
Iets van: 
"Multifuctionele parameters: een enkele parameterdeclaratie zorgt voor type-checking, correcte documentatie, etc" ?
Maar ik ben het met je eens dat het niet heel makkelijk is om het kort en bondig op te schrijven zonder dat het vaag wordt.

Reply:
Mja dat is het probleem, ik wil in het Nederlands niet méér schrijven dan in het oorspronkelijke Engels, en het is de bedoeling om beknopt te zijn op die plaats...